test/algorithms/set_operations/difference/Jamfile

```diff
@@ -18,8 +18,8 @@ test-suite boost-geometry-algorithms-difference
     :
     [ run difference.cpp                    : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE : algorithms_difference ]
     [ run difference_multi.cpp              : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE : algorithms_difference_multi ]
-    [ run difference.cpp                    : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_difference_alternative ]
-    [ run difference_multi.cpp              : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_difference_multi_alternative ]
+    [ run difference.cpp                    : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE : algorithms_difference_alternative ]
+    [ run difference_multi.cpp              : : : <define>BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE : algorithms_difference_multi_alternative ]
```

Comment:
No, please don't do this!
We support rescaling for 2 more releases and should unit test it.

This mechanism is designed for this...

Reply:
OK, I see, I revert back my changes and only add the new test case that is not tested with rescaling since it is failing.

Reply:
I guess your comment is addressed so I can merge now.